openshift-hack/e2e/annotate/rules.go

```diff
@@ -111,7 +110,40 @@ var (
 			`Netpol \[LinuxOnly\] NetworkPolicy between server and client using UDP should enforce policy based on Ports`,
 			`Netpol \[LinuxOnly\] NetworkPolicy between server and client using UDP should enforce policy to allow traffic only from a pod in a different namespace based on PodSelector and NamespaceSelector`,
 
+			// The new NetworkPolicy test suite is extremely resource
+			// intensive and causes itself and other concurrently-running
+			// tests to be flaky.
+			// https://bugzilla.redhat.com/show_bug.cgi?id=1980141
+			`\[sig-network\] Netpol `,
```

Comment:
These are new tests? Or, the tests are expected to be prefixed with [sig-...] from now on to be disabled? Or, these will appear after the 1.27 rebase lands?

Reply:
Those tests and rules are copied from origin, see [Move k8s-specific rules to our fork](https://github.com/openshift/origin/pull/27570/commits/a29317caa522e29749dba899997e4ef03f2fd71f) commit in origin PR, previously we had a mix of rules in here and in origin. This PR splits the rules so they live where the test is.